xonsh/completers/xompletions.py

```diff
@@ -0,0 +1,46 @@
+import xonsh.xontribs as xx
+from xonsh.tools import color_style_names
```

Comment:
This should be `import xonsh.tools as xt` to get the benefits of lazy module amalgamation.